bentoml/artifact/fasttext_model_artifact.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from bentoml.artifact import BentoServiceArtifact, BentoServiceArtifactWrapper
+from bentoml.exceptions import MissingDependencyException
+
+try:
```

Comment:
Looks like this is causing the unit tests to fail. Because this file got loaded whenever `bentoml/artifact/__init__.py` was imported. We are planning to change how these artifact classes handle optional dependencies, but as for now, could you change it to importing fasttext within the load/save function?  similar to how the xgboost artifact is implemented: https://github.com/bentoml/BentoML/blob/master/bentoml/artifact/xgboost_model_artifact.py

Reply:
Thanks for the suggestion, I've updated the artifact to import later on.